spyder/widgets/shortcuts.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please name this file as `shortcutssummary.py` because just `shortcuts` is a bit vague :-)